src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public enum Colors {
```

Comment:
Enum shouldnt be named in plural form
```suggestion
public enum Color {
```